README.md

```diff
@@ -535,8 +535,8 @@ Set to true to disable fqdn SSL cert check
 Default: False
 
 ##### `ssl_mode`
```

Comment:
Shouldn't we support both to keep backwards compatibility otherwise this is then a breaking change?
Do you think they will accept this upstream?
Thoughts?

Reply:
I thought of and started doing it, but it gets to messy with conditions.
So I thought for lower versions we'd use previous releases and use latest for new versions

Reply:
Ok, that means we diverge from upstream and can't submit to upstream either.
If you are fine with that then that's fine by me - hopefully, short term and upstream add this support at some point or we rework this later when we have some time.

Reply:
I'll raise a ticket to do the work separately